docs/writing-docs/mdx.md

```diff
@@ -104,7 +104,7 @@ Typically, when you use Storybook MDX, you define stories in the MDX and documen
 
 If you don't define stories in your MDX, you can write MDX documentation and associate it with an existing story, or embed that MDX as its own documentation node in your Storybook's navigation.
 
-If you don't define a Meta, you can write Markdown and associate with an existing story. See ["CSF Stories with MDX Docs"](https://github.com/storybookjs/storybook/blob/master/addons/docs/docs/recipes.md#csf-stories-with-mdx-docs).
+If you don't define a Meta, you can write Markdown and associate with an existing story. See ["CSF Stories with MDX Docs"](../../addons/docs/docs/mdx.md).
```

Comment:
@pascalduez actually we could do better, we can point this to the recipes md file. 

Turning this into "["CSF Stories with MDX Docs"](../../addons/docs/docs/recipes.md)."

As it would be a more suited place for it, don't you agree?

Reply:
Definitely, I didn't see it.

Reply:
So the opposite has been done in #12279 :woman_shrugging: